tests/tests_geomstats/test_geometric_median.py

```diff
@@ -0,0 +1,36 @@
+"""Methods for testing the Geometric Median Estimators."""
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.spd_matrices import SPDMatrices, SPDMetricAffine
+from geomstats.learning.geometric_median import WeiszfeldAlgorithm
+
+ROOT2 = gs.sqrt(2)
+
+
+class TestGeometricMedianEstimation(geomstats.tests.TestCase):
```

Comment:
These tests do not respect the new test structure: @lpereira95 what do you recommend here? Any chance you could guide Shubham with this, or should we just merge this PR and you will take care of this?

Reply:
I would say to keep it this way for now, because learning algorithms tests were not updated yet.